src/test/resources/features/DepartureFE/AmendDeclaration.feature

```diff
@@ -33,9 +33,9 @@ Feature: End to end journey for amending a departure declaration
     And I should see COMPLETED status for route details on the 'Declaration summary' page
 
     And I click on the Edit transport details link on the 'Declaration summary' page
-    And I click the Change link for Do you want to add a method of payment for transport charges? on the Transport details 'Check your answers' page
```

Comment:
This should always go to the check your answers page. If it hasn't its because the declaration json is missing the new fields. The way this is now written is when some of the data is missing so the journey automatically goes to that screen.